_posts/2020-06-24-sourcify-faq.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+published: true
+title:  "All you need to know about Sourcify"
+date:   2020-06-24
+author: Franziska Heintel
+category: Explainers
+---
+
+Welcome to Sourcify's first short FAQ! If your questions around source verification haven't been answered after reading 
+this post, please feel free to drop by the [Sourcify Gitter channel](https://gitter.im/ethereum/sourcify) and ask us 
+any question there. Also stay tuned on more turorials and developer focused content to follow here and on the [Remix blog](https://medium.com/remix-ide)!
+
+## Source Verification
+
+### What is source verification?
+
+In a software development context, the term source verification describes the process of verifying the legitimacy of code 
+before interacting with it. Solidity contract source code can be verified by checking whether the on-chain deployed code 
+is matching with published source code files. In order to allow for source verification, 
+the source and metadata files need to be published, thus enabling developers to verify that the deployed bytecode is an exact match to the 
+published files. By doing so, it can be verified that the contract code is exactly the same as when it was deployed on the
+blockchain and has not been adjusted or tampered with before deploying it.
+
+### Why is source verification important?
+
+Without open source contracts it is hard to tell what the application you are using will actually be doing. 
+One of the decentralized web's biggest aspriations is to build a trustless infrastructure, aiming to reduce the need to 
+trust third parties and service providers and decreasing censorship opportunities. It should, by design, not be required to 
+"blindly" trust a dapp developer or web3 project. It shall, in the best case, always be possible to verify it for yourself, 
```

Comment:
From our perspective always, but there's also still closed source applications out there. Not sure if we want to be so strict?

Reply:
I mean, I understand the meaning, but the expression caught my eye because it seems a little contradictory :) I'd say that a closed source application requires some trust by design so the previous sentence already makes a firm statement. If we're saying that, I see no reason not to say "always" here.

Reply:
Right, makes sense.